src/pages/events/skills.tsx

```diff
@@ -48,13 +49,30 @@ const TeamSkillsTab: React.FC<TeamSkillsTabProps> = ({ event }) => {
 
   const isLoading = isLoadingTeams || isLoadingSkills;
 
+  const filteredTeams = useMemo(() => {
+    if (!filter) return teams;
+
+    return teams?.filter(
+      (team) =>
+        team.number.startsWith(filter) ||
+        team.team_name?.toUpperCase().includes(filter)
+    );
+  }, [filter, teams]);
+
   return (
     <>
       <section className="contents">
         <Spinner show={isLoading} />
+        <input
```

Comment:
Looks like we are getting a bit of z-fighting here when scrolling the virtualized list. I think setting a z-index here would fix this visual bug.

![image](https://github.com/user-attachments/assets/6c230f49-d700-4e41-b451-03f0ebfb20d9)